vnext/Microsoft.ReactNative.Managed/JSValueGenerator.cs

```diff
@@ -0,0 +1,488 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using static System.Linq.Expressions.Expression;
+
+namespace Microsoft.ReactNative.Managed
+{
+    static class JSValueGenerator
```

Comment:
>     [](start = 0, length = 4)

Please keep formatting of this file the same as for others. It shows as a new file because the indentation changed from two characters to four.